CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 ## Unreleased Changes
 
 ### Breaking Changes
-`--Xvalidators-builder-registration-default-gas-limit` is removed in favour of `--validators-builder-registration-default-gas-limit`
+- `--Xvalidators-builder-registration-default-gas-limit` is removed in favour of `--validators-builder-registration-default-gas-limit`
```

Comment:
maybe we should list `-Xp2p-sync-rate-limit` as no longer available just in case anyone uses it. could potentially reference this PR removing it.... not sure anyone is using it but it'll be confusing if they are...

Reply:
Yeah, good point, added it to CHANGELOG. Not sure how diligent we should be with `--X` option since users should be aware they could be changed at any point, but I suppose it is good to inform the users anyways.